observability/micrometer/src/test/java/org/wildfly/extension/micrometer/SubsystemParsingTestCase.java

```diff
@@ -115,48 +103,36 @@ public void testSchema() throws Exception {
         super.testSchema();
     }
 
+    @Override
+    public void testSubsystem() throws Exception {
+        standardSubsystemTest(null);
```

Comment:
If you put all the expressions in micrometer.xml you can then delete all the other test methods except this one and testSchema. And delte micrometer-expressions.xml All the other tests duplicate what standardSubsystemTest does.

Reply:
@bstansberry Done. If you wouldn't mine verifying that I deleted only the tests you had in mind, I'd appreciate it. :)

Reply:
@jasondlee Actually you can delete testSchema and testSubsystem as well. It turns out both are @Test methods in the superclass, and here in testSchema you just call the superclass and in the testSubsystem you do directly what the superclass does. So no need to override the superclass.

Reply:
Ugh. I should have caught that. Changes incoming.